### ISSUE TITLE
conda install dashit ?

### ISSUE THREAD

**Issue by tseemann** [czbiohub/dashit#23]
dashit is very complicated to install, needing c++, go, python3 and ancillary binary tools.
can you please make a conda recipe for it all?
bioconda is the de facto standard system for bioinformtics software these days.
https://bioconda.github.io/contributor/index.html

**Comment by andersgs** [comment]
Given the recent support of the Chan-Zuckerberg Initiative, I would expect all czbiohub tools to be on bioconda: https://chanzuckerberg.com/eoss/proposals/reproducibility-in-bioinformatics-by-sustaining-bioconda-development/

We are happy to help if needed... And, it will ensure a wider audience are able to take advantage of the tools.

**Comment by EmperorDali** [comment]
Yes, we agree - a conda package is a priority. I tried my hand at an initial package, but couldn't successfully get it to build, seemingly because of the compiler requirements:

`meta.yaml` https://gist.github.com/EmperorDali/ac61e29b4bfcbfb75b90d2f5ccab1df3

`build.sh` https://gist.github.com/EmperorDali/8e9cf6be34b42f73b570e36a55245917

Feedback is appreciated.

**Comment by andersgs** [comment]
Let me play around with these for a little bit. I think you are missing some important flags.

**Comment by EmperorDali** [comment]
@andersgs any luck with the conda side of things?

In the meantime, I've added a Docker image with DASHit installed, which makes getting going pretty easy. Info in `README.md`
